File: _config/lua/role_db.lua
local role_db = {
  types = { 'I', 'S', 'S', 'S', 'S', 'I', 'S', 'F', 'B', },
  fields = { 'id', 'name', 'icon', 'quality', 'attr', 'order', 'upId', 'heigh', 'isHero', },
  values = {
    { '1', '雷古曼', 'icon/head16', 'Text/q1', 'Text/attr_huo_1', '16', '1', '12.3', 'True', },
    { '2', '雷古曼-轻装', 'icon/head17', 'Text/q2', 'Text/attr_huo_1', '17', '2', '43.2', 'False', },
    { '3', '雷古曼-重装', 'icon/head18', 'Text/q3', 'Text/attr_huo_1', '18', '3', '23.234', 'True', },
    { '4', '雷古曼-铠甲兽龙', 'icon/rob6', 'Text/q3', 'Text/attr_huo_1', '', '511', '65.3', '', },
  },
}
return role_db

File: Assets/wumai/Scripts/Data/role_db.cs
using System.Collections.Generic;
using LuaInterface;

public class role_db : DbBase
{
    public string name = null;
    public string icon = null;
    public string quality = null;
    public string attr = null;
    public string upId = null;

    public int id = 0;
    public int order = 0;

    public float heigh = 0f;

    public bool isHero = false;


    private static Dictionary<int, role_db> m_allData = new Dictionary<int, role_db>();
    public override void init(string id, LuaTable db)
    {
        var key = int.Parse(id);
        var data = db[key] as LuaTable;
        this.id = int.Parse(data["id"].ToString());
        this.name = (string)data["name"];
        this.icon = (string)data["icon"];
        this.quality = (string)data["quality"];
        this.attr = (string)data["attr"];
        this.order = int.Parse(data["order"].ToString());
        this.upId = (string)data["upId"];
        this.heigh = float.Parse(data["heigh"].ToString());
        this.isHero = bool.Parse(data["isHero"].ToString());
        m_allData.Add(this.id, this);
    }

    public role_db this[int key]
    {
        get{
        role_db db;
        m_allData.TryGetValue(key, out db);
        return db;}
    }

}